layout/mainmenu/mapselector/selector.xml

```diff
@@ -9,406 +9,404 @@
 
 	<MomentumMapSelect>
 		<Panel id="MapSelectorContents" class="mapselector">
-			<Panel id="MapSelectorLeft" class="mapselector__left">
-				<Panel class="mapselector__left-wrapper">
-					<Panel class="mapselector-category-tabs">
-						<RadioButton
-							id="CategoryListMaps"
-							class="mapselector-category-tabs__tab"
-							group="CategoryListNavBar"
-							selected="true"
-							onactivate='$("#CategoryContents").SetHasClass("mapselector__category-contents--playlists", false)'
-						>
-							<Panel class="flow-right horizontal-align-center">
+			<Panel id="MapSelectorLeftContainer" class="mapselector__left-container">
+				<Panel id="MapSelectorLeft" class="mapselector__left">
+					<Panel class="mapselector__left-wrapper">
+						<Panel class="mapselector-category-tabs">
+							<RadioButton
+								id="CategoryListMaps"
+								class="mapselector-category-tabs__tab"
+								group="CategoryListNavBar"
+								selected="true"
+								onactivate='$("#CategoryContents").SetHasClass("mapselector__category-contents--playlists", false)'
+							>
+<Panel class="flow-right horizontal-align-center">
 								<Image
-									class="mapselector-category-tabs__image"
-									src="file://{images}/panorama-variant.svg"
-									textureheight="32"
-									style="padding: 1px;"
-								/>
+										class="mapselector-category-tabs__image"
+										src="file://{images}/panorama-variant.svg"
+										textureheight="32"
+										style="padding: 1px;"
+									/>
 								<Label class="text-h mapselector-category-tabs__text" text="#MapSelector_Maps" />
 							</Panel>
 						</RadioButton>
 						<RadioButton
-							id="CategoryListPlaylists"
-							class="mapselector-category-tabs__tab"
-							group="CategoryListNavBar"
-							onactivate='$("#CategoryContents").SetHasClass("mapselector__category-contents--playlists", true)'
-						>
-							<Panel class="flow-right horizontal-align-center">
-								<Image class="mapselector-category-tabs__image" src="file://{images}/playlist-play.svg" textureheight="32" />
-								<Label class="text-h mapselector-category-tabs__text" text="#MapSelector_Playlists" />
-							</Panel>
-						</RadioButton>
-					</Panel>
+								id="CategoryListPlaylists"
+								class="mapselector-category-tabs__tab"
+								group="CategoryListNavBar"
+								onactivate='$("#CategoryContents").SetHasClass("mapselector__category-contents--playlists", true)'
+							>
+<Panel class="flow-right horizontal-align-center">
```

Comment:
weird indentation here

Reply:
Weird that Prettier+Husky didn't catch this...